### DISCUSSION TITLE
Lets stay connected by following each other

### DISCUSSION THREAD

**Discussion by Mattral** [drknzz/GitHub-Achievements#256] [Q&A]
Hello follow and drop a message then I will follow you back

**Comment by aishasalim** [comment]
Yes! Please!

**Reply by the1Riddle** [reply]
please follow me back, i follow you..
thanks

**Reply by Mattral** [reply]
I followed you, please follow back and connect

**Comment by the1Riddle** [comment]
follow me, and i will follow you back.
